src/instructlab/sdg/__init__.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Third Party
+from datasets import Dataset
+
+# Local
+from .pipeline import Pipeline
+
+
+class SDG:
```

Comment:
Should this be in the `__init__.py` file?

Reply:
Do you have a suggestion for refactor? didnt want to create another sdg class

Reply:
ideally not ... i'd probably put it in an sdg.py and then import it here

Reply:
though I'll probably put that off for future changes

Reply:
@shivchander My recommendation is to just have a `classes.py` file and then in this file you'd just add a `from .classes import SDG`